crates/layout/src/layout.rs

```diff
@@ -380,13 +380,9 @@ impl LayoutManager {
     ) {
         if let Some(fixed_width) = fixed_width {
             style.min_size.width = taffy::prelude::Dimension::Points(fixed_width as f32);
-            style.max_size.width = taffy::prelude::Dimension::Points(fixed_width as f32);
-            style.size.width = taffy::prelude::Dimension::Points(fixed_width as f32);
```

Comment:
Did this end up fixing the wrapping issue?

Reply:
No, I remember adding this toward the beginning when I was. having some layout problems. This fixed my issue at the time but it didn't seem right, so I think it's good to remove, as I didn't see any regressions when I removed it.